modules/standard/AutoMath.chpl

```diff
@@ -1394,6 +1488,26 @@ module AutoMath {
     return yn(n.safeCast(c_int), x);
   }
 
+  // When removing this deprecated function, be sure to remove chpl_yn and
+  // move its contents into Math.chpl to reduce the symbols living in this
+  // module
+  pragma "no doc"
+  pragma "last resort"
+  deprecated "yn is no longer included by default, please 'use' or 'import' the 'Math' module to call it"
+  inline proc yn(n: int, x: real(32)): real(32) {
+    return chpl_yn(n, x);
+  }
+
+  // When removing this deprecated function, be sure to remove chpl_yn and
+  // move its contents into Math.chpl to reduce the symbols living in this
+  // module
+  pragma "no doc"
+  pragma "last resort"
+  deprecated "yn is no longer included by default, please 'use' or 'import' the 'Math' module to call it"
```

Comment:
I had a similar question last time and I don't think I totally understood the answer, so sorry to be duplicating this one:

This deprecation message says it _is_ no longer included by default, but that doesn't seem right to me because it is still included by default right now, but won't be in the future.

It seems like a present-tense makes more sense here, something like "this function will no longer be included by default in the near future" (bad wording :), but the idea is there) seems to more accurately indicate what is happening.

Reply:
Heh, fair :) I'll improve the deprecation warning, thanks!

Reply:
How does it look now?